cms/urls.py

```diff
@@ -72,6 +72,7 @@
         r'^course_info_update/{}/(?P<provided_id>\d+)?$'.format(settings.COURSE_KEY_PATTERN),
         'course_info_update_handler'
     ),
+    url(r'^home/$', 'course_listing', name='home'),
```

Comment:
I don't see that the old URL is removed. Should it be? Or is that handling the redirect?